src/game/gameOverModal.js

```diff
@@ -0,0 +1,24 @@
+import styles from "./gameOverModal.scss";
+
+const modal = document.createElement("div");
```

Comment:
It is `const` and out of any function. Change it to `SCREAMING_SNAKE_CASE` and call it `MODAL` for consistency.

Reply:
done